cpp/dolfin/io/VTKWriter.cpp

```diff
@@ -246,7 +246,7 @@ void write_ascii_mesh(const mesh::Mesh& mesh, int cell_dim,
        << "ascii"
        << "\">";
   for (int types = 0; types < num_cells; types++)
-    file << vtk_cell_type << " ";
+    file << int(vtk_cell_type) << " ";
```

Comment:
Is this cast needed?

Reply:
Without the cast, this yields:
```
<DataArray  type="UInt8"  Name="types"  format="ascii">F F </DataArray>
```
For vtk_cell_type=70

Reply:
Declare it as `int` in the first place, or try `std::to_string`.

The `int` cast is something that someone could easily remove later because it's not obvious why it's needed.

Reply:
It is best not to write non-string data to `std::cout` the results can be... unexpected. There is `boost::lexical_cast` and the slightly more basic but still fine `std::to_string`.